applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml

```diff
@@ -2,7 +2,7 @@ prompt-proto-service:
 
   podAnnotations:
     # Update this field if using latest or static image tag in dev
-    revision: "1"
+    revision: "30"
```

Comment:
I think this change could have been done purely in Argo, once the changes in "Update LATISS dev configs" were pushed.